README.md

```diff
@@ -1,12 +1,18 @@
 # News Site
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Create a News Site using flexbox to layout the page. Create a responsive design with the following views; Desktop 4 columns, tablet 2 columns, mobile 1 column.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Overall i think i started out the way that i planned not to. I was afraid to ask questions, i had no plan and i kind of started with all sections at once. I started over again (over and over again) and everytime i added styling in CSS it became more confusing and complex.
+
+After all this panicing i started with a "mobile first" approach. Focusing on making the header first. Using flexbox i styled it in three different ways depending on which screen you use. 
```

Comment:
Sounds good with mobile-first approach and taking on a chunk at a time to not be overwhelmed 👍